2023/did-wg.html

```diff
@@ -337,13 +315,17 @@ <h2>Success Criteria</h2>
 more implementations interoperating with each other. In order to advance to
 Proposed Recommendation, each normative specification must have an open
 test suite of every feature defined in the specification.</p>
+
+    <p>In order for <a href="#deliverable-did-resolution">DID Resolution</a> to advance to <a href="https://www.w3.org/Consortium/Process/#RecsPR" title="Proposed Recommendation">Proposed Recommendation</a>, it is expected that each of the independant implementations mentioned above support at least two DID methods with an open specification (i.e. a specification that is accessible to all for implementation and deployment). It is also expected that each pair of the independant implementations mentioned above support at least one common DID method with an open specification.</p>
```

Comment:
This should set a more specific bar than just "open specification". @cwilso and I suggested over [email](https://lists.w3.org/Archives/Public/public-did-wg/2023Nov/0017.html) that it be something like

```suggestion
    <p>In order for <a href="#deliverable-did-resolution">DID Resolution</a> to advance to <a href="https://www.w3.org/Consortium/Process/#RecsPR" title="Proposed Recommendation">Proposed Recommendation</a>, it is expected that each of the independant implementations mentioned above support at least two DID methods with open specifications at at least the Candidate Recommendation Snapshot level, or a level with equivalent vetting and patent grants at another standards body. It is also expected that each pair of the independant implementations mentioned above support at least one common DID method with an open specification.</p>
```

Reply:
We would oppose this. 

None of the DID methods are at that level of maturity. 

We also find it disingenuous to introduce this 6 minutes before the end of the call to resolve this phase of the FO process.